### ISSUE TITLE
rtmps facebook

### ISSUE THREAD

**Issue by lordmen99** [starrtc/starrtc-android-demo#25]
hi team, its can direct push to rtmps facebook? if can its will be coolest  sdk :+1:

**Comment by lordmen99** [comment]
i try push to rtmp facebook url. still not working, can u help me how to do that?

**Comment by elesos** [comment]
first you must have a rtmp server(like nginx rtmp),  then you can use our sdk to push rtmp stream to your rtmp server. 
finnally you open the rtmp url in some video players like vlc etc.

**Comment by lordmen99** [comment]
yes, i know that. now how add nginx rtmp in server side , so can push rtmp from android app when live streaming. can u share some code, so when i doing rtmp streeaming from app, then example i enable push to facebook its will push streaming into facebook
